homeassistant/components/nest/config_flow.py

```diff
@@ -240,72 +249,77 @@ async def async_step_device_project(
             errors=errors,
         )
 
-    async def async_step_pubsub(
```

Comment:
We no longer need this stage because now if no topics exist, we can just create them.